### PR TITLE
Add .aab installation support

### PR DESCRIPTION
**Pull request by nghiaviminh**
Drag & drop installation and launching of .aab files. Set your own keystore options to sign APKs in storage-temp unit

### CODE REVIEW

File: lib/util/bundletool.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+
+var Promise = require('bluebird')
+var yauzl = require('yauzl')
+
+var logger = require('./logger')
+
+module.exports = function(options) {
+  return new Promise(function(resolve, reject) {
+    var defaultBaseDir = ('/tmp/bundletool')
+    var bundletoolUrl = 'https://api.github.com/repos/google/bundletool/releases/latest'
```
Comment (koral--) [inline lib/util/bundletool.js:14]: IMO bundletool should be bundled with docker image.
STF may be deployed in various environments which may not have network access.

**Review by koral-- — CHANGES_REQUESTED** [review]

**Review by koral-- — APPROVED** [review]
LGTM, but your branch is outdated.
Could you merge/rebase on latest changes?

**Comment by nghiaviminh** [comment]
Of course, I'll update this branch on latest changes

**Comment by nghiaviminh** [comment]
I've rebased the branch, you can merge changes.

**Pull request merged**